osh/word_eval.py

```diff
@@ -1060,10 +1060,28 @@ def _Nullary(self, val, op, var_name, vsub_token, vsub_state):
                     result = value.Str('')
```

Comment:
this might need `type: value_t` because it's the first one

That is, it infers the specific type `value.Str`, but you may have to declare the parent type

Reply:
Yeah, thanks.